driver-embedded/src/main/com/mongodb/embedded/client/MongoClients.java

```diff
@@ -46,7 +50,8 @@ public static synchronized void init(final MongoEmbeddedSettings mongoEmbeddedSe
             mongoEmbeddedLibrary = MongoEmbeddedCAPI.create(mongoEmbeddedSettings.getYamlConfig(),
                     mongoEmbeddedSettings.getLogLevel().toCapiLogLevel(), mongoEmbeddedSettings.getLibraryPath());
         } catch (Exception e) {
-            throw new MongoClientEmbeddedException("The mongo embedded library could not be initialized", e);
+            throw new MongoClientEmbeddedException(format("The mongo embedded library could not be initialized%n"
+                    + "Server error message: %s", e.getMessage()), e);
```

Comment:
Adding the server error message to the error message turns out to be incredibly insightful,

Reply:
You shouldn't have to do this.  By making `e` the cause of MongoClientEmbeddedException, the message from `e` should be shown when MongoClientEmbeddedException is thrown (which usually will cause a call to `#printStackTrace`).

Reply:
That maybe but ultimately, it is much more helpful this way.  Especially in the cases where we provide hints to the cause (eg: Make sure the client is closed first) because sometimes they aren't the actual cause of the exception.